docs/databags.md

```diff
@@ -93,7 +93,10 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `['bag']` - A string that points to an externalized Apps Hash in which all keys (except this one) are valid. 
 * `[app]` - The name of an app to manage (disk name)
```

Comment:
I like it. Added to my PR changes branch (which will be squash merged after everything): 0c767d53791f107ca83f0a826f9b22bd67a7c551